_content/monographs/pali-literature-3-1-publication_malasekera-gp-bode-mabel-haynes-saddhatissa-hammalawa.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Pali literature : a 3-in-1 publication"
+authors:
+  - "G.P. Malalasekera"
+  - "Mabel Haynes Bode"
+  - "Hammalawa Saddhātissa"
+external_url: "https://www.bps.lk/olib/bp/bp628h_Pali-Literature-of-Asia.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1Z33p4BU6jaFA0x1CJKV-RyTeIMS75qy8/view?usp=sharing"
+course: theravada-roots
+tags:
+  - pali-literature
+  - sea
+  - sri-lankan-roots
+  - burma
```

Comment:
The `burma` tag is for secular items. `burmese-roots` would be the tag for the history of Burmese Buddhism.